test/cutting/test_cutting_workflows.py

```diff
@@ -213,11 +203,9 @@ def test_reconstruction_with_samplerv2():
         num_samples=100,
     )
 
-    # Use BackendSamplerV2 with AerSimulator, following
-    # https://github.com/Qiskit/qiskit-aer/issues/2078#issuecomment-1971498534
+    # Use SamplerV2 in local mode with AerSimulator
     samplers = {
-        label: BackendSamplerV2(backend=AerSimulator())
-        for label in subexperiments.keys()
+        label: SamplerV2(backend=AerSimulator()) for label in subexperiments.keys()
```

Comment:
Any particular reason for choosing `AerSimulator` over [GenericBackend](https://docs.quantum.ibm.com/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2)?

Reply:
I did not consider it; I was just following https://docs.quantum.ibm.com/api/migration-guides/local-simulators#aersimulator, which uses `AerSimulator` as a backend.

Reply:
OK thanks, it looks like it's just strictly more performant than the statevector stuff in Qiskit